docker-compose.yml

```diff
@@ -12,14 +12,18 @@ services:
     image: redis:alpine
     ports:
       - "6379:6379"
-  selenium-firefox:
+  selenium:
     image: b4handjr/selenium-firefox
-    volumes:
-      - .:/send
-    working_dir: /send
-    expose:
-      - "4444"
     ports:
-      - "5900"
+      - "5900:5900"
```

Comment:
I'd rather keep this to just whatever open port can be found. I feel like using a specific port could cause problems.

Reply:
we could make it an env variable, then anyone who does have problems on the default port can change it. Dynamic is less convenient in the common case.